test/functional/Java14andUp/src/org/openj9/test/java/lang/Test_Thread.java

```diff
@@ -0,0 +1,203 @@
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+import org.testng.annotations.Test;
+import org.testng.AssertJUnit;
+
+/**
+ * This test is to verify matching behavior to the RI from the following implementation change
+ * discussed in the Java 14 release notes:
+ * 
+ * "The specification for java.lang.Thread::interrupt allows for an implementation to only track the 
+ * interrupt state for live threads, and previously this is what occurred. As of this release, the 
+ * interrupt state of a Thread is always available, and if you interrupt a thread t before it is started, 
+ * or after it has terminated, the query t.isInterrupted() will return true."
+ */
+
+@Test(groups = { "level.sanity" })
+public class Test_Thread {
+
+    /* interrupt thread before it has started */
+    @Test
+    public void test_interruptBeforeRun() {
+        String name = "interruptBeforeRun: ";
+        Thread t = new Thread();
+
+        /* Verify thread is dead and has not been interrupted */
+        AssertJUnit.assertFalse(name + "thread should not be alive", t.isAlive());
+        AssertJUnit.assertFalse(name + "interrupt flag should not be set", t.isInterrupted());
+
+        t.interrupt();
+
+        /* Verify that the thread was successfully interrupted. */
+        AssertJUnit.assertTrue(name + "thread that has not started should have interrupt flag set", t.isInterrupted());
+    }
+
+    /* interrupt thread after it has ended */
+    @Test
+    public void test_interruptAfterRun() throws Throwable {
+        String name = "interruptAfterRun: ";
+        Thread t = new Thread();
+
+        /* run thread */
+        t.start();
+        t.join();
+
+        /* Verify thread is dead and has not been interrupted */
+        AssertJUnit.assertFalse(name + "thread should not be alive", t.isAlive());
+        AssertJUnit.assertFalse(name + "interrupt flag should not be set", t.isInterrupted());
+
+       t.interrupt();
+
+        /* Verify that the thread was successfully interrupted. */
+        AssertJUnit.assertTrue(name + "thread that has ended was was interrupted", t.isInterrupted());    
+    }
+
+    /* if thread is interrupted before start, verify that flag is still set after run. */
+    @Test
+    public void test_interruptAtStartSetAfterRun() throws Throwable {
+        String name = "interruptAtStartSetAfterRun: ";
+        Thread t = new Thread();
+
+        /* Verify thread is dead and has not been interrupted */
+        AssertJUnit.assertFalse(name + "thread should not be alive", t.isAlive());
+        AssertJUnit.assertFalse(name + "interrupt flag should not be set", t.isInterrupted());
+
+        t.interrupt();
+
+        /* run thread */
+        t.run();
```

Comment:
This and the join() don't do anything, so this is really the same test as test_interruptBeforeRun().